safe-global/safe-wallet-web
# [Seedless Onboarding] Account center redesign

## What it solves

Resolves #2589

## How this PR fixes it

- Adds a new component `SocialLoginInfo` that is similar to `EthHashInfo`
- Refactors `useConnectWallet`

## How to test it

1. Open the app
2. Press Connect Wallet
3. Observe the new design
4. Connect a MM wallet
5. Observe the new design
6. Connect via Social Login
7. Observe the new design
8. The copy button should copy the address
9. The explorer button should open the address in the block explorer

## Screenshots
<img width="325" alt="Screenshot 2023-10-10 at 12 30 44" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/dbef6dda-4be5-4ee6-9568-17a409b1e7a7">
<img width="345" alt="Screenshot 2023-10-10 at 12 32 44" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/12923890-15aa-450d-bd98-77de4505b5e6">
<img width="348" alt="Screenshot 2023-10-10 at 12 33 01" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/19de4fd0-91fa-41d7-9e8a-3dfbc640395a">
<img width="597" alt="Screenshot 2023-10-10 at 16 36 20" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/a78ba683-e5cb-4478-849b-9c547a0a303e">

## Checklist
